process-automation/latest/admin/release.md

```diff
@@ -147,6 +147,20 @@ The **Undeploy** action removes a deployed application.
 
 The **Manage Permissions** option allows for additional users to be granted user or administrator access to an application without having to upgrade it.
 
+### Monitoring
+
+You can monitor the health of your installation by using the Monitoring dashboard. There is a dashboard for each deployed application.
+
+To access the dashboard.
```

Comment:
I guess this should end with a colon

Reply:
I see this is done in a mixed way with a colon for bullet lists and a dot for ordered lists... never mind then.